apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-launch-sidebar.tsx

```diff
@@ -43,7 +45,13 @@ registerPlugin( 'a8c-editor-site-launch', {
 
 		return (
 			<LocaleProvider localeSlug={ window.wpcomEditorSiteLaunch?.locale }>
-				<LaunchContext.Provider value={ { siteId: window._currentSiteId } }>
+				<LaunchContext.Provider
+					value={ {
+						siteId: window._currentSiteId,
+						flow: FLOW_ID,
+						redirectTo: redirectToWpcomPath,
+					} }
```

Comment:
TypeScript error about the missing required properties `locale` and `openCheckout`